README.md

```diff
@@ -26,7 +26,7 @@ Read in a different language: [![CN](/assets/flags/CN.png)**CN**](/README.chines
 
 # Latest Best Practices and News
 
-- **🇯🇵 Japanese translation:** Our guide is now also translated to Janapense! Courtesy of the amazing [YukiOta](https://github.com/YukiOta) and [Yuta Azumi](https://github.com/YA21)
+- **🇯🇵 Japanese translation:** Our guide is now also translated to Japanense! Courtesy of the amazing [YukiOta](https://github.com/YukiOta) and [Yuta Azumi](https://github.com/YA21)
```

Comment:
@andrewjbarbour  Looks like there's also a need to fix the fix. 😁  Should be `Japanese` ✌️

Reply:
Thanks for catching this, @pmventura! :)